src/migrations/20231123111823-restructure-national-centers.js

```diff
@@ -0,0 +1,54 @@
+const { prepMigration } = require('../lib/migration');
```

Comment:
what's with the name of this file?

Reply:
Do you mean it should be removed?

Reply:
20231123111823-restructure-national-centers.js

Reply:
oops